msg/distance_sensor.msg

```diff
@@ -5,7 +5,7 @@ uint64 timestamp		# time since system start (microseconds)
 float32 min_distance		# Minimum distance the sensor can measure (in m)
 float32 max_distance		# Maximum distance the sensor can measure (in m)
 float32 current_distance	# Current distance reading (in m)
-float32 covariance		# Measurement covariance (in m), 0 for unknown / invalid readings
+float32 covariance		# Measurement covariance (in m^2), 0 for unknown / invalid readings
```

Comment:
Unrelated change, but can you also rename this to variance instead?

Reply:
Should this be done in this PR?

Reply:
I've done it.